devsetup/scripts/standalone.sh

```diff
@@ -102,6 +103,14 @@ parameter_defaults:
     NovaEnableRbdBackend: false
 __EOF__
 
+cat >\$HOME/sriov_template.yaml <<__EOF__
+parameter_defaults:
+    NovaPCIPassthrough:
+      - devname: "dummy-dev"
+        physical_network: "dummy_sriov_net"
+    NeutronPhysicalDevMappings: "dummy_sriov_net:dummy-dev"
```

Comment:
wondering what creates these dummy NIC in standalone deployment?

Reply:
perhaps we will need something on the rdo side? https://github.com/rdo-infra/rdo-jobs/blob/9733a0297fe2076616a99eee5de40ffa904d5ea4/playbooks/data_plane_adoption/setup_tripleo_os_net_config.yaml#L53